src/emulator/modules/SceProcessmgr/SceProcessmgr.h

```diff
@@ -19,6 +19,12 @@
 
 #include <module/module.h>
 
+struct VitaTimeval
```

Comment:
This file is auto-generated so the struct declaration will get erased when regenerating module files.

It might be better to put it in `SceProcessmgr.cpp` as it's only used in that one place at the moment. `SceProcessmgr.cpp` is also auto-generated, but at least everything will be in one place.

Reply:
>`SceProcessmgr.cpp` is also auto-generated

How does that work, how does code stay there then?

Reply:
Good point. I added documentation in https://github.com/Vita3K/Vita3K/wiki/Module-generation, copied from #20.

Sorry, I forgot to do that sooner.